docs/retrieving-stats.rst

```diff
@@ -9,13 +9,31 @@ First, when running Locust with the web UI, you can retrieve CSV files under the
 Secondly, you can run Locust with a flag which will periodically save the CSV file. This is particularly useful
 if you plan on running Locust in an automated way with the ``--no-web`` flag::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -t10m
+    locust -f examples/basic.py --csv=example --no-web -n10 -t10m
 
 or for v0.8 (where the ``-t`` option isn't available)::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -c10
+    locust -f examples/basic.py --csv=example --no-web -n10 -c10
 
 You can also customize how frequently this is written if you desire faster (or slower) writing::
 
     import locust.stats
     locust.stats.CSV_STATS_INTERVAL_SEC = 5 # default is 2 seconds
+
+This data will look like::
+
+    $cat example_distribution.csv
+    "Name","# requests","50%","66%","75%","80%","90%","95%","98%","99%","100%"
+    "GET /",31,4,4,4,4,4,4,4,4,4
```

Comment:
isn't the "Method" field header missing here?

Reply:
I'm not sure, it's identical to what is generated by locust for the stats pages currently.

Reply:
oh I see.  Method and Name are merged into a "Name" column.  This seems like a bug to me (though unrelated to your PR